src/common/PresentationData.ts

```diff
@@ -70,3 +71,16 @@ export interface ComparedModule {
   pcntOfSession?: number,
   status?: string,
 }
+export interface ComparedCalledModule {
+  callerID: number,
+  calleeID: number,
+  callerModuleName: string,
+  calleeModuleName: string,
+  callerTimesCalled: number,
+  calleeTimesCalled: number,
```

Comment:
callerTimesCalled should be changed to timesCalled, calleeTimesCalled to calleeTotalTimesCalled (rename those diff fields as well).